Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -6,24 +6,93 @@ class Program
     {
         static void Main(string[] args)
         {
-            Test("42*47=1?74", 9);
-            Test("4?*47=1974", 2);
-            Test("42*?7=1974", 4);
-            Test("42*?47=1974", -1);
-            Test("2*12?=247", -1);
-            Console.ReadKey(true);
+            //Console.WriteLine("Main");
+            Test("42*47=1?74");
+            Test("4?*47=1974");
+            Test("42*?7=1974");
+            Test("42*?47=1974");
+            Test("2*12?=247");
+            //Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
-            var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+        private static void Test(string args)
+        {
+            //Console.WriteLine(args);
+            Console.WriteLine(FindDigit(args));
+        }
+
+        public static int FindDigit(string equation)
+        {
+            //Console.WriteLine("FindDigit");
+            int i=0, index=-1, star=-1;

```

Comment:
Rename variables with some meaning full names, also move core function to a new class "FixMultiplication" as directed in problem statement. It will also help you to understand class invocations.